fvm/evm/emulator/state_test.go

```diff
@@ -88,7 +88,7 @@ func (s *storageTest) run(runner func(state *gethState.StateDB)) error {
 
 	runner(state)
 
-	s.hash, err = state.Commit(true)
+	s.hash, err = state.Commit(0, true)
```

Comment:
@ramtinms @janezpodhostnik there is a breaking change in this function. I added the `0` to be able to build, but you should update it to the correct value.